README.md

```diff
@@ -218,9 +296,79 @@ res.render("login", {
       size,
       color,
     }),
-  card: Card({
-    title: "Login With Ory",
-    children: Message({ message: "Woah there", severity: "info" }),
+  card: UserAuthCard({
+    title: !(flow.refresh || flow.requested_aal === "aal2")
+      ? "Sign In"
+      : "Two-Factor Authentication",
+    flow: flow as SelfServiceFlow,
+    flowType: "login",
+    cardImage: "ory-logo.svg",
+    additionalProps: {
+      forgotPasswordURL: "recovery",
+      signupURL: initRegistrationUrl,
+      logoutURL: logoutUrl,
+    },
   }),
 })
 ```
+
+---
+
+### Component System
+
+Elements solely rely on React components since they are easy to write and
+provides support to a large React based ecosystem. The project then bundles
+these components to their respective needs. An example is bundling for Preact
+which you can find in the
+[packages](https://github.com/ory/elements/tree/main/packages) folder. It uses
+the react components directly in this case but bundles it specifically for
+Preact support.
+
+Each component relies on a some CSS, which are located in the
+[theme](https://github.com/ory/elements/tree/main/src/theme) directory. To
+understand how this works, please refer to the [CSS System](#css-system)
+
+#### ExpressJs systems
+
+Express is an edge-case which requires the component to be wrapped by
+`ReactDOMServer` to produce only HTML. Each component needs to be wrapped by
```

Comment:
```suggestion
Express JS has limited support for React components, by only rendering the markup to HTML without the dynamic elements of a component. Each component needs to be wrapped by
```

Reply:
hmm, i mean it is still "dynamic" - it just isn't dynamic on the client side. So the props can be injected still on server-side and the html produced will still change.

Reply:
Hmm, yes, you're right... What's a good word for all the hooks, then?